pyproject.toml

```diff
@@ -1,6 +1,40 @@
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
+requires = ["setuptools>=69", "wheel", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "canopen"
+authors = [
+    {name = "Christian Sandberg", email = "christiansandberg@me.com"},
+]
+description = "CANopen stack implementation"
+readme = "README.rst"
+requires-python = ">=3.6"
+license = {file = "LICENSE.txt"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Intended Audience :: Developers",
+    "Topic :: Scientific/Engineering",
+]
+dependencies = [
+    "python-can >= 3.0.0",
+]
+dynamic = ["version"]
+
+[project.urls]
+documentation = "https://canopen.readthedocs.io/en/stable/"
+repository = "https://github.com/christiansandberg/canopen"
+
 [tool.setuptools_scm]
-write_to = "canopen/_version.py"
+version_file = "canopen/_version.py"
+
+[tool.pytest.ini_options]
+testpaths = [
+    "test",
+]
+filterwarnings = [
+    "ignore::DeprecationWarning",
```

Comment:
Could this be useful to see anyway?

Reply:
Maybe so.  I suggest we leave it be until after the release.  The only warnings I get currently are for `datetime.utcnow()` and the `bustype` argument to `can.interface.Bus`.